packages/whats-new/src/style.scss

```diff
@@ -131,7 +132,8 @@ $wpcom-modal-footer-height: 30px + ( $wpcom-modal-footer-padding-v * 2 );
 .whats-new-page__visual {
 	height: 40%;
 	padding: 25px 25px 0;
-	background: #1581d8;
+	// background: #1581d8;
+	background: var( --studio-blue );
```

Comment:
Is `--studio-blue` the correct theme variable to use or is there a different like `--color-primary`?